es/lessons/basics/strings.md

```diff
@@ -0,0 +1,158 @@
+---
+version: 1.1.1
+title: Cadenas
+---
+
+Cadenas, listas de caracteres, Grafemas y puntos de código.
+
+{% include toc.html %}
+
+## Cadenas
+
+Las cadenas en Elixir no son mas que una secuencia de bytes. Vamos a ver un ejemplo:
+
+```elixir
+iex> string = <<104,101,108,108,111>>
+"hello"
+iex> string <> <<0>>
+<<104, 101, 108, 108, 111, 0>>
+```
+
+Concatenando la cadena con el byte `0` IEx muestra la cadena como un binario porque ya no es una cadena válida. Este truco puede ayudarnos a ver los bytes subyacentes de cualquier cadena.
```

Comment:
Also here, _bytes_ should be in _italic_

Reply:
Done